assets/section-featured-blog.css

```diff
@@ -7,21 +7,50 @@
 }
 
 .blog .placeholder {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 22rem;
-  text-align: center;
-  padding: 4rem 2rem 5rem;
-  margin: 0 2rem;
+  margin: 0 1.5rem;
 }
 
 @media screen and (min-width: 750px) {
+  .blog .placeholder {
+    text-align: center;
+  }
+}
+
+@media screen and (min-width: 990px) {
   .blog .placeholder {
     margin: 0;
   }
 }
 
+.blog .placeholder .placeholder-content {
+  padding: 3rem;
+}
+
+.blog .placeholder .placeholder-image {
+  background: rgba(var(--color-foreground), .04);
+  overflow: hidden;
+  position: relative;
+  padding-bottom: 66.6%;
+}
+
+.blog .placeholder .placeholder-svg {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  fill: currentColor;
+}
```

Comment:
This look very similar to `media` component class. Could we leverage that along with `.media--landscape`?

Reply:
Using .media and .media--landscape, does save a couple of lines here and makes sense to utilize. I currently still need to include the centering styles though (see https://github.com/Shopify/dawn/pull/152#discussion_r669793547)